src/appmixer/kafkadavid/platform/ListOpenConnections/ListOpenConnections.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const connections = require('../../connections');
```

Comment:
There is a security issue, you went around the node cache by storing the connections to the `process`, but by calling this component, every user will be able to get all Kafka connections from all users in Appmixer.

Reply:
@martin-krcmar how would an end-user get this? Do you mean by using the "JavaScript Function" modifier? How can an end-user get access to the `process`?

Reply:
I mean by adding this component ListOpenConnections to their flow. OnStart -> LIstOpenConnections -> SendEmail and you get to your mailbox all open connections.

Reply:
@martin-krcmar right, the `ListOpenConnections` is not going out, it was just a handy debugging tool. It's removed now.